### ISSUE TITLE
[🐛 BUG]: Prefetch option is ignored when jobs pipelines are created via RPC

### ISSUE THREAD

**Issue by rauanmayemir** [roadrunner-server/roadrunner#1197]
### No duplicates 🥲.

- [X] I have searched for a similar issue in our bug tracker and didn't find any solutions.

### What happened?

`Prefetch` option is ignored when creating jobs pipelines via RPC:

```
$this->jobs->create(
    new AMQPCreateInfo(
        name: "pipeline_1",
        prefetch: 1,
        queue: "queue_1",
        exchange: 'amqp.direct',
        routingKey: "queue_1",
        multipleAck: false,
        requeueOnFail: false,
        durable: true
    )
);
```

<img width="969" alt="image" src="https://user-images.githubusercontent.com/9691/177279667-40815e62-12e0-4544-ab7b-b2af9a4093c7.png">

### Version (rr --version)

2.10.4

**Comment by rustatian** [comment]
@rauanmayemir Hello 👋🏻 

Yep, confirmed. Fix will be released on Thursday (07.07.2022).
The reason is in an incorrect string to int conversion.

**Comment by rauanmayemir** [comment]
Confirming this is fixed. 🔥
